lib/hexpm_web/templates/policy/tos.html.md

```diff
@@ -1,81 +1,107 @@
 
-You retain your rights to any Content you submit, post or display on or through the Services. By submitting, posting or displaying Content on or through the Services, you grant us a worldwide, non-exclusive, royalty-free license to use, copy, reproduce, process, adapt, modify, publish, transmit, display and distribute such Content in any and all media or distribution methods (now known or later developed). Any Content you submit or post to the Services may be stored by Hex and will be subject to these terms indefinitely or until Hex, at its sole discretion, terminates the Content.
+You retain your rights to any Content you submit, post or display on or through the Services. Except in respect of Private Content (defined in Section 8 below), by submitting, posting or displaying Content on or through the Services for public access, you grant us a worldwide, non-exclusive, royalty-free license to use, copy, reproduce, process, adapt, modify, publish, transmit, display and distribute such Content in any and all media or distribution methods (now known or later developed). Any Content you submit or post to the Services may be stored by Six Colors and will be subject to these terms indefinitely or until Six Colors, at its sole discretion, terminates the Content.
 
-You are responsible for your use of the Services, for any Content you provide, and for any consequences thereof, including the use of your Content by other users and third party services. You understand that your Content may be syndicated, broadcast, distributed, or published by third party services and if you do not have the right to submit Content for such use, it may subject you to liability. Hex will not be responsible or liable for any use of your Content by Hex in accordance with these Terms. You represent and warrant that you have all the rights, power and authority necessary to grant the rights granted herein to any Content that you submit.
+You are responsible for your use of the Services, for any Content you provide, and for any consequences thereof, including the use of your Content by other users and third party services where such content has been published publicly via the Services. You understand that your Content (other than Private Content) may be syndicated, broadcast, distributed, or published by third party services and if you do not have the right to submit Content for such use, it may subject you to liability. Six Colors will not be responsible or liable for any use of your Content by Six Colors in accordance with these Terms. You represent and warrant that you have all the rights, power and authority necessary to grant the rights granted herein to any Content that you submit.
```

Comment:
```suggestion
You are responsible for your use of the Services, for any Content you provide, and for any consequences thereof, including the use of your Content by other users and third party services where such content has been published publicly via the Services. You understand that your Content (other than Private Content) may be syndicated, broadcast, distributed, or published by third party services and if you do not have the right to submit Content for such use, it may subject you to liability. Six Colors will not be responsible or liable for any use of your Content by Six Colors in accordance with these Terms. You represent and warrant that you have all the rights, power, and authority necessary to grant the rights granted herein to any Content that you submit.
```